packages/zero-client/src/client/context.ts

```diff
@@ -192,4 +214,16 @@ export class ZeroContext implements QueryDelegate {
       }
     }
   }
+
+  queryHistograms(hash: string): HistogramsMap {
+    const histogram = this.#materializeHistograms.get(hash);
+    if (histogram === undefined) {
+      return new Map();
+    }
+    return new Map([['materialize', histogram]]);
```

Comment:
why is this a map? 🤔 Does it only have 2 entries? `materialize` and `advance`?

Reply:
I was expecting there to be more histograms per query. I think removing it now would be fine.